src/Sylius/Behat/Context/Api/Shop/AddressBookContext.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Shop;
+
+use ApiPlatform\Core\Api\IriConverterInterface;
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Component\Core\Model\AddressInterface;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Repository\AddressRepositoryInterface;
+use Webmozart\Assert\Assert;
+
+final class AddressBookContext implements Context
```

Comment:
Do we need this new context? On address book api we use `Context/Api/Shop/AddressContext.php` i guess

Reply:
@SirDomin you can change context `Context/Api/Shop/AddressContext.php` to `AddressBookContext ` and merge both 😄

Reply:
Well, this way we got different files for different address contexts i'd stay with this solution

Reply:
I would vote for merging these two contexts into `AddressBookContext`